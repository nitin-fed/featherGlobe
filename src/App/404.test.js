import React from 'react'
import renderer from 'react-test-renderer'
import MissingPage from './404'


describe('404', () => {
    test('Should render', () => {
        const component = renderer.create(<MissingPage  />)
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
        
    })
});

