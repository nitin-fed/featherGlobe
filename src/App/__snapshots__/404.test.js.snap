// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 Should render 1`] = `
<div>
   Missing Page
</div>
`;
